news/2019-09-07-community-mentorship-program-fall-2020-mentee-signups-now-open.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2020 Signups Now Open"
+date: 2019-09-08 12:50:00 +0000
+---
+
+The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping, or expanding your current knowledge and skills, this program will serve as a great opportunity for you!
```

Comment:
```suggestion
The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping or expand your current knowledge and skills, this program will serve as a great opportunity for you!
```

Reply:
current wording works better here, sounds more natural

Reply:
you're trying to say that the reader wants to expand their knowledge, right? in that case u want normal present tense instead of continuous

you need these two bracketed parts to match tense:
"wanting to [get into beatmapping] or [expand your current knowledge and...]"

Reply:
okay sure why not